pyanaconda/modules/common/base.py

```diff
@@ -24,7 +24,7 @@
 from pyanaconda.core.async_utils import run_in_loop
 from pyanaconda.core.timer import Timer
 from pyanaconda.dbus import DBus
-from pyanaconda.task import publish_task
+from pyanaconda.modules.common.task import publish_task
```

Comment:
I'm still thinking that `task` is not descriptive. Cloud you rename it in process to `installation_task` or similar please?

Or we can do that in special PR. I'll leave that on you.

Reply:
But we might use tasks for all kinds of long running jobs.

Reply:
Hmm true. I still don't like task name but I'm not able to find something better so I guess it is fine then.